### PR TITLE
Update configuration for redirect before routing

### PR DESCRIPTION
**Pull request by Jonathan2022Bausch**
With the current displayed configuration redirecting before routing is not operable

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,14 +60,12 @@ a link.
...
 
-    Neos:
-      Flow:
-        http:
-          chain:
-            process:
-              chain:
-                redirect:
-                  position: 'before routing'
+Neos:
```
Comment (Sebobo) [inline README.md:63]: The indentation is required for displaying it as code, or which would be even better to wrap it with ```yaml to produce correct highlighting.

**Comment by Sebobo** [comment]
I don't understand this change. The default setting is `after routing` and this entry in the readme is for people who have problems with that. And your change would adjust both to be the same?

File: README.md (hunk 1 above)
Comment (Jonathan2022Bausch) [reply README.md:64]: Good suggestion, thank you

**Comment by Jonathan2022Bausch** [comment]
> I don't understand this change. The default setting is `after routing` and this entry in the readme is for people who have problems with that. And your change would adjust both to be the same? Or is it actually about the yaml path which is different?

@Sebobo Thank you for the catch! I had to correct the yaml path, but the indication "before" would obviously be the intended one.

**Review by Sebobo — APPROVED** [review]
Great thx! We forgot to adjust this since after the introduction of middlewares.

**Pull request merged**